assets/_scss/pages/_index.scss

```diff
@@ -106,11 +110,115 @@ body {
 
   .widget-header {
     margin-bottom: 1em;
-    text-align: center;
+  }
+}
+
+.slider-input {
+  -webkit-appearance: none;
+  width: 100%;
+  margin: 4.2px 0;
+  max-width: none;
 
-    h2 {
-      font-size: 1.5em;
+  &:focus {
+    outline: none;
+
+    &::-webkit-slider-runnable-track {
+      background: lighten($color-gray-cool-light, 3);
+    }
+
+    &::-ms-fill-lower {
+      background: lighten($color-gray-cool-light, 3);
     }
+    &::-ms-fill-upper {
+      background: lighten($color-gray-cool-light, 3);
+    }
+  }
+
+  &::-webkit-slider-runnable-track {
+    width: 100%;
+    height: 7.6px;
+    cursor: pointer;
+    box-shadow: 0px 0px 1px rgba(0, 0, 0, 0), 0px 0px 0px rgba(13, 13, 13, 0);
+    background: lighten($color-gray-cool-light, 3);
+    border-radius: 1.3px;
+  }
+
+  &::-webkit-slider-thumb {
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+    border: 1px solid rgba(95, 95, 95, 0.4);
+    height: 16px;
+    width: 16px;
+    border-radius: 50px;
+    background: #ffffff;
+    cursor: pointer;
+    -webkit-appearance: none;
+    margin-top: -4.4px;
+  }
+
+  &::-moz-range-track {
+    width: 100%;
+    height: 7.6px;
+    cursor: pointer;
+    box-shadow: 0px 0px 1px rgba(0, 0, 0, 0), 0px 0px 0px rgba(13, 13, 13, 0);
+    background: lighten($color-gray-cool-light, 3);
+    border-radius: 1.3px;
+  }
+
+  &::-moz-range-thumb {
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+    border: 1px solid rgba(95, 95, 95, 0.4);
+    height: 16px;
+    width: 16px;
+    border-radius: 50px;
+    background: #ffffff;
+    cursor: pointer;
+  }
+
+  &::-ms-track {
+    width: 100%;
```

Comment:
Properties should be ordered background, border-color, color, cursor, height, width